src/client-side-metrics/exporter.ts

```diff
@@ -157,15 +157,16 @@ function getMetric(
 ) {
   const streaming = dataPoint.attributes.streaming;
   const app_profile = dataPoint.attributes.app_profile;
+  const status = dataPoint.attributes.status;
   return {
     type: metricName,
     labels: Object.assign(
       {
         method: dataPoint.attributes.method,
         client_uid: dataPoint.attributes.client_uid,
-        status: dataPoint.attributes.status,
```

Comment:
If `status: undefined` is included for application latencies then the MetricServiceClient call will fail with an error so this step is necessary.

Reply:
The reason for this is because application_latencies doesn't include status, right?

So status on line 169 will be null, meaning it won't be sent? Or am I reading this wrong?

Just want to make sure I understand why status would be undefined here